docs/ru/mkdocs.yml

```diff
@@ -37,7 +37,7 @@ plugins:
     data: data
 nav:
 - FastAPI: index.md
-- Languages:
+- Языки:
```

Comment:
IMO, it shouldn't be translated. If someone ends up by an accident on russian version of documentation, it could be difficult to change it. Also, I guess, it's a convention - other language versions of documentation don't translate it.

Reply:
OK. But I will wait for more comments so as not to increase commits.

Reply:
Ok You are right. I will change it in the next PRs.